modules/rostests/winetests/msvcrt/locale.c

```diff
@@ -636,6 +639,9 @@ static void test_setlocale(void)
         len = sprintf(buf, "%d", GetACP());
     ret = setlocale(LC_ALL, ".ACP");
     ok(ret != NULL, "ret == NULL\n");
+#ifdef __REACTOS__
+    if (ret == NULL) ptr = NULL; else
```

Comment:
Are the crashes you would get by not adding these two ifdefs REACTOS, due to a yet-unimplemented case in our (U)CRT ? Or is it due to a bug in the winetest itself?

Reply:
This tests MSVCRT, not UCRT.